eng/pipelines/templates/steps/install-dotnet.yml

```diff
@@ -12,11 +12,20 @@ steps:
       msbuildArguments: /p:WorkFolder="$(Agent.WorkFolder)" /p:BuildDirectory="$(Agent.BuildDirectory)"
   # Installation steps need to be uncommented when switching to a newer SDK that's not available on DevOps agents
   - task: UseDotNet@2
-    displayName: 'Use .NET Core SDK'
+    displayName: 'Use .NET SDK'
     retryCountOnTaskFailure: 3
     inputs:
       useGlobalJson: true
       performMultiLevelLookup: true
+  - task: UseDotNet@2
+    condition: and(succeeded(), ne(variables['Agent.OS'], 'Windows_NT')) # Windows supports MultiLevelLookup and doesn't need explicit framework installation
+    displayName: 'Use .NET 6.0 SDK'
+    retryCountOnTaskFailure: 3
+    inputs:
+      # AspNetCore runtime pack can't be installed outside of SDK and we need it for intergation tests
```

Comment:
```suggestion
      # AspNetCore runtime pack can't be installed outside of SDK and we need it for integration tests
```

Reply:
That came along for the ride, actually, but if I don't need to make any more changes I'll do this in a separate PR just to avoid resetting this one (since random failures tend to pop up and these checks take forever).